stdlib/src/builtin/int_literal.mojo

```diff
@@ -269,6 +269,47 @@ struct IntLiteral(
         """
         return self
 
+    @always_inline("nodebug")
+    fn __divmod__(self, rhs: Self) -> Tuple[Self, Self]:
+        """Return the quotient and remainder of the division of self by rhs.
+
+        Args:
+            rhs: The value to divide on.
+
+        Returns:
+            The quotient and remainder of the division.
+        """
+        var quotient: Self = self.__floordiv__(rhs)
+        var remainder: Self = self - (quotient * rhs)
+        return quotient, remainder
+
+    @always_inline("nodebug")
+    fn __round__(self, ndigits: Int) -> Self:
```

Comment:
I can't see tests for this or its counterpart in `Int`. Maybe forgot to push it?

Reply:
Added here: https://github.com/msaelices/mojo/commit/6039f4e21db55f795608f5faf378c8e035f4679e